backend/googlephotos/googlephotos.go

```diff
@@ -134,14 +134,22 @@ rclone mount needs to know the size of files in advance of reading
 them, so setting this flag when using rclone mount is recommended if
 you want to read the media.`,
 			Advanced: true,
-		}},
+		},
+			{
```

Comment:
Can you use the same bracketty style as the other options please? So `}, {` and `}},`